main/src/vtr_path_planning/include/vtr_path_planning/mpc/custom_steam_evaluators.hpp

```diff
@@ -0,0 +1,101 @@
+// Copyright 2021, Autonomous Space Robotics Lab (ASRL)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * \file custom_steam_evaluators.hpp
+ * \author Jordy Sehn, Autonomous Space Robotics Lab (ASRL)
+ */
+
+#pragma once
+
+#include "steam/evaluable/state_var.hpp"
+
+namespace steam {
+namespace stereo {
```

Comment:
Based on how you appear to be using this, I don't think it should be in the namespace stereo anymore.

Reply:
fair, this was just legacy of the code I used as a template to write these evaluators - removed stereo namespace and updated relevant code in latest commit